bodhi/tests/server/functional/test_admin.py

```diff
@@ -0,0 +1,44 @@
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import copy
+from webtest import TestApp
+
+from bodhi.server import main
+from bodhi.server.models import (User, Group)
```

Comment:
Whoops, yeah i had it in there for an earlier iteration of my tests when i was hacking on it, where i was using it. Forgot to take it out. Not sure why i didnt notice the test failing when I ran my final tests before pushing though :/ 

thanks for picking this up! cheers!